### PR TITLE
Vc/multi shares

### PR DESCRIPTION
**Pull request by vadiminc**
Fixes #<issue_number_goes_here>

> It's a good idea to open an issue first for discussion.

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

File: src/commands/actions/arguments/index.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import keystoreArgument from './keystore';
```
Comment (IlyaVi) [inline src/commands/actions/arguments/index.ts:1]: export { default as <name> } from ./<name';

then use as 
import { <name1>, <name2>, ... } from './arguments';

File: src/lib/KeyShares/KeySharesData/validators/index.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { OpeatorsListValidator } from './operator-unique';
```
Comment (IlyaVi) [inline src/lib/KeyShares/KeySharesData/validators/index.ts:1]: same here

**Review by IlyaVi — APPROVED** [review]

**Review by alexstolr — APPROVED** [review]
lgtm

**Review by axelrod-blox — APPROVED** [review]

File: src/lib/KeyShares/KeyShares.ts [hunk 3]
```diff
@@ -183,30 +68,27 @@ export class KeyShares {
...
-      version: `v${pkg.version}`,
-      createdAt: new Date().toISOString(),
-      data: this.data || null,
-      payload: this.payload.readable || null,
-    }, null, '  ');
-  }
+    // Using a helper function to process each item
+    const processItem = async (item: any) => {
+      const keySharesItem = new KeySharesItem();
+      await keySharesItem.fromJson(item);
```
Comment (meshin-dev) [inline src/lib/KeyShares/KeyShares.ts:76]: From json should be static class method and return class instance.

File: src/lib/KeyShares/KeyShares.ts [hunk 4]
```diff
@@ -183,30 +68,27 @@ export class KeyShares {
...
-    const partsArr = [];
-    for (let i = 0; i < parts; i++) {
-      const start = i * partLength;
-      const end = start + partLength;
-      partsArr.push(arr.slice(start, end));
+    if (Array.isArray(body.shares)) {
+      // Process each item in the array
+      for (const item of body.shares) {
+        const processedItem = await processItem(item);
+        this.shares.push(processedItem);
```
Comment (meshin-dev) [inline src/lib/KeyShares/KeyShares.ts:84]: this.shares.push(KeySharesItem.fromJson(item));

**Review by meshin-dev — APPROVED** [review]
Please check comments

**Pull request merged**